great_expectations/datasource/fluent/schemas/SparkAzureBlobStorageDatasource/CSVAsset.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "CSVAsset",
-    "description": "Base model for most fluent datasource related pydantic models.\n\nAdds yaml dumping and parsing methods.\n\nExtra fields are not allowed.\n\nSerialization methods default to `exclude_unset = True` to prevent serializing\nconfigs full of mostly unset default values.\nAlso prevents passing along unset kwargs to BatchSpec.\nhttps://docs.pydantic.dev/usage/exporting_models/",
+    "description": "Mixin encapsulating logic specific to regex-based FilePathDataAssets",
```

Comment:
these doc string changes are unfortunate, so i'll look into fixing it. they weren't correct in the first place, so at least we're not losing any data.